selfdrive/car/honda/carstate.py

```diff
@@ -62,6 +63,7 @@ def get_can_signals(CP, gearbox_msg):
       ("SCM_FEEDBACK", 10),
       ("SCM_BUTTONS", 25),
     ]
+  assert any(msg == main_on_sig_msg for msg, _ in checks)
```

Comment:
can parser already does this

Reply:
```suggestion
```

Reply:
The intention of this assert is to confirm that the passed-in `main_on_sig_msg` is represented in the list of checks.  The CAN parser can't possibly know about our local args.

In other words, we don't want someone to pass `main_on_sig_msg="FOO"` later on, and have no check that "FOO" messages are occurring.

We can't naively append `checks += [main_on_sig_msg, 10]`, because both `SCM_FEEDBACK` and `SCM_BUTTONS` need to have checks, and the rates depend on model-- see above.

So I think this assert is relevant and concisely provides some safeguard.

Reply:
the can parser doesn't need to know about the args, it uses the list of signals. i.e. if you pass in FOO, it'll complain that FOO isn't in checks. see https://github.com/commaai/opendbc/blob/master/can/parser_pyx.pyx#L81

Reply:
oh, that's great-- thank you for explaining!